ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java

```diff
@@ -279,7 +298,11 @@ private void initSpatials() {
         for (Path path : levelGenerator.getPathPieces(COMMANDER_LOCATION)) {
             addDisplayable(path);
         }
-
+        for (PlatformPosition position : PlatformPosition.values()) {
+        	Carrier carrier = createCarrier(position);
+        	addEntity(carrier);
+        }
+        
```

Comment:
This causes the carriers to spawn even when players are not connected, please fix this.